### ISSUE TITLE
Define errors and replace `Exception` with them

### ISSUE THREAD

**Issue by eggplants** [shadowmoose/selenium_drivers#18]
This package often uses `Exception` class when raising an error. But I suggest to replace them with more precise, detail, and internal Error classes like `pyderman.Error.BrowserError` to make easy to handle properly.

**Comment by shadowmoose** [comment]
Good idea. Generally any error raised signifies that this library has failed, but it would certainly be better to add more context there.
